example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -199,9 +199,11 @@ async function ingestAndPublishGranule(config, testSuffix, testDataFolder, publi
     testDataFolder
   );
 
-  ingestAndPublishGranuleExecutionArns.push(await buildAndExecuteWorkflow(
+  const { executionArn } = await buildAndExecuteWorkflow(
```

Comment:
Yeah, this wasn't great.    May want to consider changes/ in #2311/rolling those in.